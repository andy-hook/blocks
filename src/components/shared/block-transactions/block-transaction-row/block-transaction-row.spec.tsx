import React from "react"
import BlockTransactionRow from "./block-transaction-row"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockTransactionRow />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockTransactionRow />)
    expect(tree).toMatchSnapshot()
  })
})
