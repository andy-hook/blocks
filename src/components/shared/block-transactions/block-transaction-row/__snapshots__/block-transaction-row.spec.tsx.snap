// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockTransactionRow /> renders correctly 1`] = `
<tr
  className="sc-bdVaJa bDWFJH"
>
  <td
    className="sc-bwzfXH sc-htpNat KtFsv"
  />
  <td
    className="sc-bwzfXH sc-bxivhb bUZGRr"
  />
  <td
    className="sc-bwzfXH sc-ifAKCX cFlEyZ"
  >
    <div />
  </td>
  <td
    className="sc-bwzfXH sc-EHOje jvzENE"
  />
</tr>
`;
